pkgs/by-name/fu/fuzzel/package.nix

```
@@ -54,14 +56,15 @@ stdenv.mkDerivation (finalAttrs: {
     tllist
     fcft
   ] ++ lib.optional enableCairo cairo
-    ++ lib.optional pngSupport libpng;
+  ++ lib.optional pngSupport libpng
+  ++ lib.optional (svgBackend == "librsvg") librsvg;
 
   mesonBuildType = "release";
 
-  mesonFlags = [
-    "-Denable-cairo=${if enableCairo then "enabled" else "disabled"}"
-    "-Dpng-backend=${if pngSupport then "libpng" else "none"}"
-    "-Dsvg-backend=${if svgSupport then "nanosvg" else "none"}"
+  mesonFlags = with lib; [
+    (mesonOption "enable-cairo" (if enableCairo then "enabled" else "disabled"))
```

```suggestion
    (mesonEnable "enable-cairo" enableCairo)
```
https://noogle.dev/f/lib/mesonEnable can do this for us

---

Ok